keras_cv/applications/object_detection/retina_net/__internal__/layers/decode_predictions.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv import layers
```

Comment:
This import is bit anti pattern, and might cause circular deps in future. Basically __interal__ lib shouldn't import the layers that might use itself.

Reply:
good point, I'll update this.